pdns/recursordist/rec-main.cc

```diff
@@ -2730,6 +2730,10 @@ static void recursorThread()
       try {
         if (!::arg()["lua-dns-script"].empty()) {
           t_pdl = std::make_shared<RecursorLua4>();
+          const std::string& path = ::arg()["lua-global-include-path"];
```

Comment:
this is, if I'm counting right, the fourth time I see this line. How about making it an argument to baselua and the derived classes?

Reply:
you mean accessing the ::arg() from BaseLua?

Reply:
I was thinking "pass a global include path"

Reply:
maybe i now understood this correctly.